pandas/tests/plotting/test_groupby.py

```diff
@@ -65,3 +66,48 @@ def test_plot_kwargs(self):
 
         res = df.groupby("z").plot.scatter(x="x", y="y")
         assert len(res["a"].collections) == 1
+
+    @pytest.mark.parametrize("column, expected_axes_num", [(None, 2), ("b", 1)])
+    @pytest.mark.parametrize("label", [None, "d"])
+    def test_groupby_hist_with_legend(self, column, expected_axes_num, label):
+        # GH 6279
+        # Histogram can have a legend
```

Comment:
could you remove this line? or combine it with `GH 6279` and the same for others?